Hazelcast.Net/Hazelcast.Core/PagingPredicate.cs

```diff
@@ -95,13 +95,13 @@ public PagingPredicate(int pageSize, IPredicate predicate = null, IComparer<KeyV
         }
 
         /// <summary>
-        /// <c>IComparer&lt;KeyValuePair&lt;object, object&gt;&gt;</c>> implementation used to sort the result on client side.
+        /// <c>IComparer</c>> implementation used to sort the result on client side.
         /// </summary>
         /// <remarks>
         /// <b>WARNING:</b> This comparer implementation should be hazelcast serializable and must have the same
         /// implementation on server side.
         /// </remarks>
-        public IComparer<KeyValuePair<object, object>> Comparer
```

Comment:
does this API change break backward compatibility since the IComparer can be provided by user in the constructor?